### ISSUE TITLE
Unable to set pod.strategy.type

### ISSUE THREAD

**Issue by fernandopn** [authelia/chartrepo#98]
Hey Folks,

I have a small instalation of authelia that uses a PVC for persistence (ReadWriteOnce).
That means that RollingUpdate will not work for mu authelia instalation (since you cannot start another job before removing the previous).

I've been trying to set .Values.pod.strategy.type but I was unsucessful. Helm complains with:
```
  Type    Reason  Age                From             Message
  ----    ------  ----               ----             -------
  Normal  info    46m (x2 over 46m)  helm-controller  HelmChart 'flux-system/networking-authelia' is not ready
  Normal  error   37m (x2 over 46m)  helm-controller  Helm install failed: template: authelia/templates/deployment.yaml:22:13: executing "authelia/templates/deployment.yaml" at <include "authelia.deploymentStrategy" .>: error calling include: template: authelia/templates/_helpers.tpl:460:18: executing "authelia.deploymentStrategy" at <.Values.pod.strategy>: nil pointer evaluating interface {}.pod
```

It seems to be related to https://github.com/helm/helm/issues/8026

Reference to my deployment: https://github.com/fernandopn/swarm

**Comment by fernandopn** [comment]
https://github.com/authelia/chartrepo/pull/99 could be a solution to it, but I have no clue how to make those checks pass.

**Comment by james-d-elliott** [comment]
closed by #101

**Comment by fernandopn** [comment]
Thanks James for working on it. My setup works like a charm!

https://github.com/fernandopn/swarm/blob/main/cluster/apps/networking/authelia/helm-release.yaml
